blocks/bricks/__init__.py

```diff
@@ -570,6 +570,44 @@ def apply_outputs(self):
         return self.application_methods[-1].outputs
 
 
+class Concatenate(Brick):
+    """Merges several variables by concatenating them.
+
+    Parameters
+    ----------
+    axis : int
+        Axis along which to concatenate inputs.
+
+    Notes
+    -----
+    This is a very straightforward wrapping of a Theano operation without
+    parameters of its own. It's mainly useful in conjunction with the
+    :class:`~blocks.filter.VariableFilter`.
```

Comment:
These two should be changed as well then: https://github.com/mila-udem/blocks/blob/master/blocks/bricks/__init__.py#L479-L488